certbot-apache/certbot_apache/configurator.py

```diff
@@ -962,6 +1069,8 @@ def _copy_create_ssl_vhost_skeleton(self, avail_fp, ssl_fp):
                 "the potential to create redirection loops.".format(avail_fp,
                                                                     ssl_fp),
                 reporter.MEDIUM_PRIORITY)
+        self.aug.set("/augeas/files%s/mtime" %(self._escape(ssl_fp)), "0")
+        self.aug.set("/augeas/files%s/mtime" %(self._escape(avail_fp)), "0")
```

Comment:
This was commited by Noah in the original PR: https://github.com/certbot/certbot/pull/3373/commits/7133f0cf8c227b11436a47d5c8e3b055325540e3

I believe the underlying reason was augeas not flagging the file for reload after changes made in rapid succession or something alike.